### ISSUE TITLE
Unable to inspect `sigma_` or `std_err_` properties after tail fit

### ISSUE THREAD

**Issue by jbogaardt** [casact/chainladder-python#156]
While these properties work just fine with the `MackChainladder`, they cannot be displayed directly once a tail is fit.
```python
import chainladder as cl
clrd = cl.load_sample('clrd')
# Fails to display
cl.TailCurve().fit_transform(clrd['CumPaidLoss'].sum()).sigma_
```
This is because the tail adds extra lags for run-off labels than exist in the underlying array.
